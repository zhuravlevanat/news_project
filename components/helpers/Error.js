import {Box} from "@material-ui/core";
// import LinearProgress from "@material-ui/core/LinearProgress";
import React from "react";
import {ThemeProvider} from "@material-ui/core/styles";
import theme from "../theme";
import Typography from "@material-ui/core/Typography";

const Error = () => (
    <Box  height='100vh' >
        <ThemeProvider theme={theme}>
            <Typography variant="h6">Something went wrong...Please, try again...</Typography>
        </ThemeProvider>
    </Box>
);

export default Error;