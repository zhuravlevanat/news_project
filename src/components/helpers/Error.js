import { Box } from "@material-ui/core";
import React from "react";
import { ThemeProvider } from "@material-ui/core/styles";
import Typography from "@material-ui/core/Typography";
import theme from "../theme";

const Error = () => (
    <Box height='100vh' >
        <ThemeProvider theme={theme}>
            <Typography variant="h6">Something went wrong...Please, try again...</Typography>
        </ThemeProvider>
    </Box>
);

export default Error;
