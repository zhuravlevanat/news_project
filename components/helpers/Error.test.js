import React from 'react';
import Error from './Error';
import renderer from 'react-test-renderer';

test('Error renders correctly', () => {
    const component = renderer.create(
        <Error />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});