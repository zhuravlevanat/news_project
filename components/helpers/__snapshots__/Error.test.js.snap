// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders correctly 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  <h6
    className="MuiTypography-root MuiTypography-h6"
  >
    Something went wrong...Please, try again...
  </h6>
</div>
`;

exports[`Layout renders correctly 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  <h6
    className="MuiTypography-root MuiTypography-h6"
  >
    Something went wrong...Please, try again...
  </h6>
</div>
`;
